mathics/builtin/numbers/calculus.py

```diff
@@ -2269,6 +2269,10 @@ def eval(self, eqs, vars, evaluation: Evaluation):
                 numer, denom = eq.as_numer_denom()
                 sympy_denoms.append(denom)
 
+        if not sympy_eqs:
```

Comment:
This breaks an existing test: `Solve[True,x]` should return `{{}}` without warnings. On the other hand,

```
Solve[{x>5}, x]                                                                                                                                                       
```
produces the message:
```
Solve::fulldim: The solution set contains a full-dimensional component; use Reduce for complete solution information.
```
and 
```
Solve[Q, x]                                                                                                                                                           
```
produces
```
Solve::naqs: Q is not a quantified system of equations and inequalities.
```

Reply:
Oh gd the warning issue was overlooked, I'll look into it.